azure-pipelines/official.yml

```diff
@@ -71,9 +71,6 @@ extends:
           suppressionFile: $(System.DefaultWorkingDirectory)\azure-pipelines\falsepositives.gdnsuppress
```

Comment:
Is the issue that repo is not cloned at all or $(Build.SourcesDirectory) variable is empty? The rest of the tasks also seems to use files from azure-pipelines so I am wondering if the change needed is to replace Build.SourcesDirectory with System.DefaultWorkingDirectory instead?

Reply:
The release jobs don't clone the repo, so `Build.SourceDirectory` is not available. Since we are only using the suppression file to exclude some files generated during the build, we can configure using the suppression file only during the build

Reply:
Sounds good but I am curious of the above tasks worked then so it may be worth experimenting later.